src/frontend/src/lib/stores/user-profile.store.ts

```diff
@@ -0,0 +1,24 @@
+import type { UserProfile } from '$declarations/backend/backend.did';
+import { writable, type Readable } from 'svelte/store';
+
+// * `undefined` means the store is not loaded yet.
+// * `null` means there was an error.
+// * `UserProfile` is the data.
+export type UserProfileStoreData = UserProfile | undefined | null;
```

Comment:
Similar to the `address.store`, don't we need the `certified: boolean;` information as well in store?

Reply:
I'll add it, it makes sense.